shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java

```diff
@@ -85,14 +85,20 @@ private void fetchConstraintsColumns(final Collection<Map<String, Object>> const
                 columns.add(column);
             }
             each.put("columns", columns);
-            Map<String, Object> parameters = new LinkedHashMap<>();
-            parameters.put("cid", each.get("oid"));
-            Collection<Object> includes = new LinkedList<>();
-            for (Map<String, Object> include : executeByTemplate(parameters, "index_constraint/%s/get_constraint_include.ftl")) {
-                includes.add(include.get("colname"));
+            getConstraintInclude(each);
+        }
+    }
+    
+    private void getConstraintInclude(final Map<String, Object> constraintsProp) {
```

Comment:
@tuichenchuxin Can we rename this method to appendConstraints?

Reply:
Thank you. Done.